doc/source/parameters/dem/boundary_conditions.rst

```diff
@@ -0,0 +1,48 @@
+Boundary Conditions
+-------------------
+In this subsection, the boundary conditions of the DEM simulation are defined. First of all, the ``number of boundary conditions`` is specified. Then for each boundary condition, its information is defined. There are four boundary types: ``fixed_wall``, ``outlet``, ``rotational`` (around the center), and ``translational``. For ``rotational`` motion, ``rotational speed`` and ``rotational vector`` are required, while for ``translational`` motion, the ``speed`` should be defined in each direction.
```

Comment:
Can you add an explanation of  what all boundary wall means and do. Like fixed_wall is a static wall, etc.

Reply:
done.